org.python.pydev.updatesite/site.xml

```diff
@@ -3,10 +3,11 @@
    <description name="PyDev Update Site" url="https://raw.githubusercontent.com/ponder-lab/Pydev/pydev_9_3_1/org.python.pydev.updatesite">
       PyDev Updatesite
    </description>
-   <feature url="features/org.python.pydev.feature_9.3.0.202203051235.jar" id="org.python.pydev.feature" version="9.3.0.202203051235">
+   <feature url="features/org.python.pydev.feature_9.3.1.qualifier.jar" id="org.python.pydev.feature" version="9.3.1.qualifier">
       <category name="org.python.pydev.updatesite"/>
    </feature>
-   <feature url="features/org.python.pydev.feature.source_9.3.0.202203051235.jar" id="org.python.pydev.feature.source" version="9.3.0.202203051235">
+   
```

Comment:
@tatianacv Please remove this space. And, let's not directly edit these XML files. Please use the UI in Eclipse for that.

Reply:
If this feature was added manually (not through the UI), then please complete remove it.